packages/graphql/jest.global-setup.js

```diff
@@ -0,0 +1,44 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const setTZ = require("set-tz");
+const neo4j = require("neo4j-driver");
+
+const TZ = "Etc/UTC";
+const INT_TEST_DB_NAME = "neo4jgraphqlinttestdatabase";
+
+module.exports = async function globalSetup() {
+    process.env.NODE_ENV = "test";
+
+    setTZ(TZ);
+
+    global.INT_TEST_DB_NAME = INT_TEST_DB_NAME;
+
+    const { NEO_USER = "admin", NEO_PASSWORD = "password", NEO_URL = "neo4j://localhost:7687/neo4j" } = process.env;
+    const auth = neo4j.auth.basic(NEO_USER, NEO_PASSWORD);
+    const driver = neo4j.driver(NEO_URL, auth);
+    const cypherCreateDb = `CREATE OR REPLACE DATABASE ${INT_TEST_DB_NAME}`;
+    const cypherDetachNodes = `MATCH (n) DETACH DELETE n`;
```

Comment:
Do we want to delete all nodes if we can't create a test database? Slightly paranoid, but might a contributor run our tests against a database which contains some data they might for some reason want to keep?

Reply:
Was thinking about this for a while as well and I have arguments for both sides. I decided on the delete all nodes approach in the end because we add a lot of nodes (1.5k or so) with each test run that it would be very annoying to clear them all up afterward. Yet I definitely see your point!
Let's NOT clear the database then, I can totally accept this solution.